dot/parachain/gossip-support/gossip_support.go

```diff
@@ -441,8 +441,45 @@ func (gs *GossipSupport) getKeyIndexAndUpdateMetrics(SessionInfo *parachaintypes
 	return authCheckResult, nil
 }
 
-func (*GossipSupport) updateGossipTopology(_ uint, _ common.Hash) error {
-	// TODO implement in #4510
+// updateGossipTopology shuffles the list of AuthorityDiscoveryID with ValidatorIndex randomly and send a
```

Comment:
```suggestion
// updateGossipTopology shuffles the list of AuthorityDiscoveryID with ValidatorIndex randomly and sends a
// [networkbridgemessages.NewGossipTopology] to overseer
```
I think the `[]` will produce a link to the type in rendered godocs.